test_dir/new_file_2

```diff
@@ -0,0 +1,6 @@
+well lets see what happens now
+
+
+well, well, well....wow man
+
+okays lets see
```

Comment:
RiflerRick committed on a restricted directory: test_dir/new_file_2in the repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir/new_file_2
Collaborators: @manasisingh67, @RiflerRick,